iocage_lib/ioc_common.py

```diff
@@ -1050,3 +1050,31 @@ def parse_package_name(pkg):
         'revision': revision,
         'epoch': epoch,
     }
+
+
+def get_host_gateways():
+    gateways = {'ipv4': {'gateway': 'none', 'interface': 'none'},
```

Comment:
Let's please stick to `None` here ( not the string ) as it was before. That is way easier to check in python and much neater imho. We can just return `"none"` from `default_gateway` function if no gateway is present.